handler/handler_test.go

```diff
@@ -101,9 +101,9 @@ func TestHealthyReady(t *testing.T) {
 func TestPush(t *testing.T) {
 	mms := MockMetricStore{}
 	mmsWithErr := MockMetricStore{err: errors.New("testerror")}
-	handler := Push(&mms, false, false, logger)
-	handlerWithErr := Push(&mmsWithErr, false, false, logger)
-	handlerBase64 := Push(&mms, false, true, logger)
+	handler := Push(&mms, false, true, false, logger)
+	handlerWithErr := Push(&mmsWithErr, false, true, false, logger)
+	handlerBase64 := Push(&mms, false, true, true, logger)
```

Comment:
A comment listing the bool arg names would help readability.

Reply:
Good point. Added.